components/management/commands/load_components.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from pathlib import Path
+
+from pydantic import ValidationError
+from django.core.management.base import BaseCommand, CommandError
+
+from blueprintapi.oscal.component import Model as ComponentModel
+from catalogs.models import Catalog
+from components.models import Component
+
+
+class Command(BaseCommand):
+    help = (
+        "Ingest component data into the Component table. "
+        "Currently assumes a single component definition defined against a single catalog"
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument("--load-test-data", type=bool, default=True)
+        parser.add_argument("--catalog", type=str, default="CMS_ARS_3_1_MODERATE")
```

Comment:
At some point I think this should be removed as an option and we should figure out what the catalog should be based on the component data.